tests/services/test_service_queryparser.py

```diff
@@ -104,16 +103,14 @@ def test_multimatch(parser, query):
         # Term is allowed and not mapped, results in a query string with the original query
         ("description:test", "description:test", "query_string", None),
         ("metadata.title:test", "metadata.title:test", "query_string", None),
-        # Terms might be mapped but not allowed, results in multi_match with the original query
-        ("title:test", "title:test", "multi_match", ["metadata.date"]),
```

Comment:
IMO, it does not make sense for the field to be mapped but not allowed. That means that the user can query it but cant at the same time. If something is defined in the mapping it will be allowed.